__tests__/app.test.js

```diff
@@ -156,4 +156,50 @@ describe("/api/articles", () => {
             });
     });
  //should I test for inaccurate count? not a 400 or 404, end user couldn't know   
+})
+describe("/api/articles/:article_id/comments", () => {
+    test("GET 200: Responds with endpoint json data", () => {
+        return request(app)
+            .get("/api/articles/3/comments")
+            .expect(200)
+            .then(({ body }) => {
+              const comments = body.comments;
+              comments.forEach((comment) => {
```

Comment:
Make sure you check the length of the array returned - if it's 0, then the .forEach won't run, therefore creating a false positive and rendering the test void! An expect just before the .forEach would be great :)